repository/test-src/org/pentaho/platform/repository2/unified/DefaultUnifiedRepositoryBase.java

```diff
@@ -115,7 +111,7 @@
 @ContextConfiguration( locations = { "classpath:/repository.spring.xml",
     "classpath:/repository-test-override.spring.xml" } )
 @SuppressWarnings( "nls" )
-public class DefaultUnifiedRepositoryBase implements ApplicationContextAware {
+public abstract class DefaultUnifiedRepositoryBase implements ApplicationContextAware {
```

Comment:
@mdamour1976 that's to accent that this class isn't a test class, it is a base for tests that use repository.